src/cmd-buildextend-metal

```diff
@@ -206,6 +206,7 @@ runvm "${target_drive[@]}" -- \
             --save-var-subdirs "${save_var_subdirs}" \
             --rootfs-size "${rootfs_size}" \
             "${luks_flag}"
+/usr/lib/coreos-assembler/finalize-artifact "${path}.tmp"
 mv "${path}.tmp" "$path"
```

Comment:
I do notice that both code paths do the `mv` after; it's probably make sense to have `finalize-artifact` do this.  But, possibly later.